### ISSUE TITLE
HuggingFace

### ISSUE THREAD

**Issue by ilikecoding777** [facebookresearch/audiocraft#139]
On the huggingface demo, it's gone. What happened?

**Comment by GrandaddyShmax** [comment]
not only that, the models are gone as well. all of them

**Comment by MavelDraconia** [comment]
Huh. Possibly a Cease and Desist from someone?

**Comment by pramatias** [comment]
same issue here. Colab notebook is not working.

**Comment by darknessNBK** [comment]
Same thing.

**Comment by KonradGmurek** [comment]
Same...

> requests.exceptions.HTTPError: 401 Client Error: Unauthorized for url: https://huggingface.co/facebook/musicgen-melody/resolve/main/compression_state_dict.bin

**Comment by charlesmccarthy** [comment]
hope this is resolved asap? same error? any alternatives?

**Comment by BetaDoggo** [comment]
Intel has the same thing. It's probably an exploit. 
![image](https://github.com/facebookresearch/audiocraft/assets/103955583/e37349d2-e9bf-4836-9431-e8d988f459c8)

**Comment by GrandaddyShmax** [comment]
Fixed the issue on my colab, (reuploaded the models), so if anyone needs musicgen feel free: https://colab.research.google.com/github/camenduru/MusicGen-colab/blob/main/MusicGen_ClownOfMadness_plus_colab.ipynb

**Comment by charlesmccarthy** [comment]
> Fixed the issue on my colab, (reuploaded the models), so if anyone needs musicgen feel free: https://colab.research.google.com/github/camenduru/MusicGen-colab/blob/main/MusicGen_ClownOfMadness_plus_colab.ipynb

Where can i get the models and how to adjust the code to reference them when running locally?

**Comment by ilikecoding777** [comment]
I used it before I posted this issue, but then after that, it now shows "404
Invalid rev id: HEAD"

**Comment by GrandaddyShmax** [comment]
> > Fixed the issue on my colab, (reuploaded the models), so if anyone needs musicgen feel free: https://colab.research.google.com/github/camenduru/MusicGen-colab/blob/main/MusicGen_ClownOfMadness_plus_colab.ipynb
> 
> Where can i get the models and how to adjust the code to reference them when running locally?

You can get this repo with everything set up:
https://github.com/GrandaddyShmax/audiocraft_plus

Or alternatively you can get the models here:
https://huggingface.co/GrandaddyShmax

**Comment by GrandaddyShmax** [comment]
> I used it before I posted this issue, but then after that, it now shows "404 Invalid rev id: HEAD" Does any of the devs know what happened on HuggingFace?

From what I understood, they and many other large groups on huggingface got hacked

**Comment by ilikecoding777** [comment]
> > I used it before I posted this issue, but then after that, it now shows "404 Invalid rev id: HEAD" Does any of the devs know what happened on HuggingFace?
> 
> From what I understood, they and many other large groups on huggingface got hacked

That's crazy. I tried the huggingface link you provided, but it keeps saying error when I use an example music file and then add a prompt. Is there any tips/tricks for MusicGen that I need to know?

**Comment by GrandaddyShmax** [comment]
> > > I used it before I posted this issue, but then after that, it now shows "404 Invalid rev id: HEAD" Does any of the devs know what happened on HuggingFace?
> > 
> > 
> > From what I understood, they and many other large groups on huggingface got hacked
> 
> That's crazy. I tried the huggingface link you provided, but it keeps saying error when I use an example music file and then add a prompt. Is there any tips/tricks for MusicGen that I need to know?

Yeah that's because my huggingface version runs on cpu which cannot run this model.. so if you want to use huggingface you'll have to duplicate the space. alternatively you can use the colab, it is always up to date

**Comment by ilikecoding777** [comment]
Oh alright. I am trying yours, and this other one that copied the model.

**Comment by pramatias** [comment]
Thanks @GrandaddyShmax. The colab version of audiocraft doesn't exactly run like the colab audiocraft+. Audiocraft+ yields a public url after the execution. 

Do we have to clone the audiocraft+ repo, run the python app.py and connect to that public url?

**Comment by GrandaddyShmax** [comment]
> Thanks @GrandaddyShmax. The colab version of audiocraft doesn't exactly run like the colab audiocraft+. Audiocraft+ yields a public url after the execution.
> 
> Do we have to clone the audiocraft+ repo, run the python app.py and connect to that public url?

if you run the repo locally, you do not need to use the public url, only the local one.
But to be honest I never ran it locally so I cannot answer it with experience, don't have a good enough gpu.

**Comment by pramatias** [comment]
> if you run the repo locally, you do not need to use the public url, only the local one.
> But to be honest I never ran it locally so I cannot answer it with experience, don't have a good enough gpu.

I run it on colab too, not locally. It just that with plain audiocraft, i ran it without gradio, just cells on the colab. In audiocraft+ i have to run the gradio on my computer, that's my question i guess. And then connect to the public url of audiocraft+ output. 

I should have made it more clear the first time.

**Comment by GrandaddyShmax** [comment]
> > if you run the repo locally, you do not need to use the public url, only the local one.
> > But to be honest I never ran it locally so I cannot answer it with experience, don't have a good enough gpu.
> 
> I run it on colab too, not locally. It just that with plain audiocraft, i ran it without gradio, just cells on the colab. In audiocraft+ i have to run the gradio on my computer, that's my question i guess. And then connect to the public url of audiocraft+ output.
> 
> I should have made it more clear the first time.

oh I see now. yeah it's different, this colab creates you link for the webui.
so if you are running colab you will need to enter that link and it will open up the webui with everything set up

**Comment by nikollax** [comment]
I'm running locally and why I have this error? "huggingface.co/facebook/musicgen-melody/resolve/main/compression_state_dict.bin"

**Comment by GrandaddyShmax** [comment]
> I'm running locally and why do I have this error? "huggingface.co/facebook/musicgen-melody/resolve/main/compression_state_dict.bin"
> 
> Also, I wanted to check with Colab, and it's the same error!
> 
> I was also looking for a file on Huggingface and it's removed for some reason.

Try using this colab: https://colab.research.google.com/github/camenduru/MusicGen-colab/blob/main/MusicGen_ClownOfMadness_plus_colab.ipynb

Or this Repo if you want to run locally: https://github.com/GrandaddyShmax/audiocraft_plus

**Comment by rsxdalv** [comment]
> I'm running locally and why do I have this error? "huggingface.co/facebook/musicgen-melody/resolve/main/compression_state_dict.bin"
> 
> Also, I wanted to check with Colab, and it's the same error!
> 
> I was also looking for a file on Huggingface and it's removed for some reason.


```
pip uninstall audiocraft
pip install git+https://git@github.com/GrandaddyShmax/audiocraft_plus@85a5112#egg=audiocraft
```

Or just edit ..\env\Lib\site-packages\audiocraft\models\loaders.py to use GrandaddyShmax repo
```
HF_MODEL_CHECKPOINTS_MAP = {
    "small": "GrandaddyShmax/musicgen-small",
    "medium": "GrandaddyShmax/musicgen-medium",
    "large": "GrandaddyShmax/musicgen-large",
    "melody": "GrandaddyShmax/musicgen-melody",
}
```

**Comment by rsxdalv** [comment]
Issue appears to be resolved and the official repository is back up again.
